.github/workflows/deploy-cloudflare.yml

```
@@ -34,49 +76,66 @@
-
-      - name: Type checking
-        run: npx tsc --noEmit
-
-      # Add more tests here as needed
-      # - name: Run unit tests
-      #   run: npm test
+          script: |
+            const deploymentUrl = '${{ steps.deploy.outputs.url }}';
+            const comment = `## 🚀 Deployment Preview
+            
+✅ **Deployment completed successfully!**
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 112: could not find expected ':'

(syntax-check)